src/modules/Y2ModuleConfig.rb

```diff
@@ -335,6 +340,33 @@ def SetDesktopIcon(file)
       true
     end
 
+    # Returns list of all profile sections from the current profile, including
+    # unsupported ones, that do not have any handler (AutoYaST client) assigned
+    # at the current system and are not handled by AutoYaST itself.
+    #
+    # @return [Array<String>] of unknown profile sections
+    def unhandled_profile_sections
+      profile_sections = Profile.current.keys
+
+      profile_handlers = @ModuleMap.map{
```

Comment:
Following our style guide, I'd use `do..end` style here. Or if you prefer, just cut down the `name` and `desc` parameter names and use an 'one-liner'.